elastic/kibana
# [Guided onboarding] Support "done" state in dropdown panel

This PR updates the "complete" guide state to align with the design. It also adds the ability for the "Continue using Elastic" button to redirect to a particular page, if a config is provided (used for the Security guide). By default, the user will be left on the current page when completing a guide.

Fixes https://github.com/elastic/kibana/issues/140976

### How to test
1. Enable guided onboarding via your `kibana.dev.yml` file: `guidedOnboarding.ui: true`
2. Start up Kibana with the example plugins enabled: `yarn start --run-examples`
3. Navigate to `/app/guidedOnboardingExample`
4. Start and complete the "Search" example guide. **Note:** We are in the process of refactoring our example guide. You will need to revert the change in https://github.com/elastic/kibana/pull/143302 in order to test the example flow E2E.
5. Verify the completion state.

### Screenshots
<img width="562" alt="Screen Shot 2022-10-12 at 1 31 38 PM" src="https://user-images.githubusercontent.com/5226211/195409945-f1e77ef2-c9d4-492e-96f9-8fd3af7d51e4.png">
